scripts/packages/BUILD

```diff
@@ -50,19 +48,15 @@ filegroup(
 filegroup(
     name = "tarpkgs",
     srcs = [
-        ":heron-api",
         ":heron-core",
-        ":heron-tools",
-        ":heron-client",
+        ":heron",
```

Comment:
Hmm. API is removed? This will affect Twitter internal use case. Is it possible for you to add it back?